prog/mutation.go

```diff
@@ -209,7 +209,7 @@ func (p *Prog) Mutate(rs rand.Source, ncalls int, ct *ChoiceTable) {
 // predicate pred.  It iteratively generates simpler programs and asks pred
 // whether it is equal to the orginal program or not. If it is equivalent then
 // the simplification attempt is committed and the process continues.
-func Minimize(p0 *Prog, callIndex0 int, pred func(*Prog, int) bool) (*Prog, int) {
+func Minimize(p0 *Prog, callIndex0 int, pred func(*Prog, int) bool, full bool) (*Prog, int) {
```

Comment:
We can rename `full` to something like `fast` (which has the opposite meaning).

In this case we can do `fast=true` minimization for crashes by default (since reproducing a crash make take significant time) and add an optional flag to `syz-repro` to use `fast=false` if we want to minimize as much as possible.

The one in `syz-fuzzer` should probably be `fast=false`.